config/core/resources/cloudauditlogssource.yaml

```diff
@@ -36,158 +36,182 @@ spec:
     kind: CloudAuditLogsSource
     plural: cloudauditlogssources
   scope: Namespaced
-  subresources:
-    status: {}
   preserveUnknownFields: false
   conversion:
     strategy: Webhook
-    webhookClientConfig:
-      service:
-        name: webhook
-        namespace: cloud-run-events
-  additionalPrinterColumns:
+    webhook:
+      # conversionReviewVersions indicates what ConversionReview versions are understood/preferred by the webhook.
+      # The first version in the list understood by the API server is sent to the webhook.
+      # The webhook must respond with a ConversionReview object in the same version it received.
+      conversionReviewVersions: ["v1beta1", "v1"]
```

Comment:
Looks like upstream has `v1` first, should we too?

Reply:
Fixed.